pandas/plotting/_matplotlib/core.py

```diff
@@ -1089,6 +1089,32 @@ def _make_plot(self):
         plot_colorbar = self.colormap or c_is_column
         cb = self.kwds.pop("colorbar", is_numeric_dtype(c_values) and plot_colorbar)
 
+        # move matplotlib.errorbar specific keywords that are not accepted by
+        # ax.scatter to an err_kwds dict for use in the call to ax.errorbar
+        ebarkeys = {
```

Comment:
I don't think hardcoding this is a robust approach as it may be brittle to API changes for `matplotlb.errorbar`.

Is there a way for this to be validate directly by `matplotlib.errorbar`